src/router/routes.ts

```diff
@@ -160,6 +160,18 @@ export const routes = [
       filingType: FilingTypes.REGISTRATION
     }
   },
+  {
+    path: '/dissolution-entity',
```

Comment:
When you say `entity` are we referring to firms specifically? 
If so, perhaps we should rename these references to indicate that, as entity is very high level, meaning it could mean any single type of business, corporation firm etc

Reply:
changed to dissolution-firm as name consistences with file/ component name. if we have multiple routes for Sp and GP we can use the same name for routes

Reply:
LGTM

Reply:
@cameron-freshworks OK to resolve this conversation?

Reply:
You bet, thanks for the update!